.checkpatch.conf

```diff
@@ -29,3 +29,4 @@
 --exclude modules/tfm/tfm/boards/src
 --exclude modules/tfm/tfm/boards/common
 --exclude doc/nrf/images
+--exclude applications/nrf5340_audio/src/utils/macros
```

Comment:
why not make code compliant instead?

Reply:
@gmarull : We have done a temporary split of the commits. Note that we will change this to a different (and more logical ) structure next week.

Reply:
The compliance issue is still there, but we now have a logic split of commits.